stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -136,6 +149,12 @@ swift_install_symlink_component(clang-resource-dir-symlink
   TARGET ../clang/${CLANG_VERSION}
   DESTINATION "lib/swift")
 
+# Adding link to "llvm" at install time.
+swift_install_symlink_component(llvm-resource-dir-symlink
+  LINK_NAME llvm
+  TARGET ../
+  DESTINATION "lib/swift")
```

Comment:
@jrose-apple Currently, libFuzzer is not actually going anywhere in LLVM install. This might be blocked by a few things. For now the goal of this PR is just to make it work with a freshly built Swift.